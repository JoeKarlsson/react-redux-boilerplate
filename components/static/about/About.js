import React from 'react';
import styles from './About.scss';

class About extends React.Component {
  render() {
    return (
      <div className={styles.About}>
        About
      </div>
    );
  }
}

export default About;
