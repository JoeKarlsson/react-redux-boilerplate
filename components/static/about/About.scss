$blue: blue;

.About {
  background-color: $blue;
}
