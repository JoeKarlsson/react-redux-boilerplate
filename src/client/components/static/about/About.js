import React from 'react';
import './About.scss';

const About = () => {
	return (
		<div className="About">
      About
		</div>
	);
};

export default About;
