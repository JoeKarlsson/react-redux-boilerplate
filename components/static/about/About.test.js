import React from 'react';
import About from './About';
import renderer from 'react-test-renderer';

//Snapshot tests
test('About page snapshot matches', () => {
  const component = renderer.create(
    <About />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
