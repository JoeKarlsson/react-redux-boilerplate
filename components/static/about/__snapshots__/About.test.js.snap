// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page snapshot matches 1`] = `
<div
  className="About"
>
  About
</div>
`;
